subprojects/performance/performance.gradle.kts

```diff
@@ -16,15 +16,17 @@ import org.gradle.gradlebuild.unittestandcompile.ModuleType
  * limitations under the License.
  */
 
-apply from: 'templates.gradle'
+apply(from = "templates.gradle")
 
 dependencies {
-    compile project(':distributionsDependencies')
+    compile(project(":distributionsDependencies"))
 
-    performanceTestImplementation libraries.commons_compress.coordinates
+    performanceTestImplementation(library("commons_compress"))
 
     // so that all Gradle features are available
-    performanceTestRuntime allTestRuntimeDependencies
+    rootProject.configurations.testRuntime.allDependencies.forEach {
```

Comment:
:x: Remove duplication of `rootProject.configurations.testRuntime.allDependencies`.